Doc/library/argparse.rst

```diff
@@ -2218,7 +2218,7 @@ support this parsing style.
 
 These parsers do not support all the argparse features, and will raise
 exceptions if unsupported features are used.  In particular, subparsers,
```

Comment:
Nit: hanging trailing comma on this line now (`In particular, subparsers, and mutually...`)

Reply:
I will change this in my next `argparse` PR, thanks! :_